src/Prevem/CoreBundle/Command/RendererPollCommand.php

```diff
@@ -18,54 +18,49 @@ protected function configure() {
       $this->setName('renderer:poll')
            ->setDescription('Create new renderer')
            ->addOption('url', NULL, InputOption::VALUE_REQUIRED, 'Provide base Url')
-           ->addOption('name', NULL, InputOption::VALUE_REQUIRED, 'Renderer name')
            ->addOption('cmd', NULL, InputOption::VALUE_REQUIRED, 'Rendering script path');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output) {
-      $params = array(
-        'scriptPath' => $input->getOption('cmd'),
-        'renderer' => $input->getOption('name'),
-        'username' => parse_url($input->getOption('url'), PHP_URL_USER),
-        'password' => parse_url($input->getOption('url'), PHP_URL_PASS),
-        'url' => $input->getOption('url'),
-      );
+      $username = parse_url($input->getOption('url'), PHP_URL_USER);
+      $password = parse_url($input->getOption('url'), PHP_URL_PASS);
```

Comment:
I like this simplification in removing `$params` (since `$params` and `$input` are basically the same thing... i.e. an open-ended bucket of whatever inputs we were given).